abi/constants.go

```diff
@@ -4,3 +4,4 @@ const trueAsByte = uint8(1)
 const falseAsByte = uint8(0)
 const optionMarkerForAbsentValue = uint8(0)
 const optionMarkerForPresentValue = uint8(1)
+const pubKeyLength = 32
```

Comment:
Promoted to a constant (for easier refactoring). If needed in the future, might be re-converted to a parameter of the serializer / codec.